system-tests/project-fixtures/angular/src/app/mount.cy.ts

```diff
@@ -164,7 +165,28 @@ describe("angular mount", () => {
     })
     cy.get('h3').contains('Hello World')
   })
-  
+
+  it('handles ngOnChanges on mount', () => {
+    cy.mount(LifecycleComponent, {
+      componentProperties: {
+        name: 'Angular'
+      }
+    })
+
+    cy.get('p').should('have.text', 'Hi Angular. ngOnInit fired: true and ngOnChanges fired: true')
+  })
+
+  it('handles ngOnChanges on mount', () => {
+    cy.mount('<app-lifecycle [name]="name"></app-lifecycle>', {
+      declarations: [LifecycleComponent],
+      componentProperties: {
+        name: 'Angular'
+      }
+    })
+
+    cy.get('p').should('have.text', 'Hi Angular. ngOnInit fired: true and ngOnChanges fired: true')
+  })
+
```

Comment:
I would add 2 testcases (template syntax, class syntax) without `componentProperties` set where we assert that `ngOnChanges` was not fired.